src/sonic-yang-models/tests/yang_model_tests/tests_config/flex_counter.json

```diff
@@ -3,7 +3,8 @@
         "sonic-flex_counter:sonic-flex_counter": {
             "sonic-flex_counter:FLEX_COUNTER_TABLE": {
                 "BUFFER_POOL_WATERMARK": {
-                    "FLEX_COUNTER_STATUS": "enable"
+                    "FLEX_COUNTER_STATUS": "enable",
```

Comment:
Where are we using this test config?

Reply:
Add test which violates the range please. Thx

Reply:
@praveen-li 
Added polling intervals to test config.
Added test cases with a valid polling interval and a test which violates the range.
Please review.